src/sentry/static/sentry/app/views/settings/projectAlerts/ruleEditor/ruleNode.jsx

```diff
@@ -145,6 +149,10 @@ const RuleNodeForm = styled('div')`
     line-height: 26px;
     min-width: 150px;
   }
+  .Select-placeholder {
+    height: 26px;
+    line-height: 26px;
+  }
```

Comment:
https://github.com/getsentry/sentry/blob/master/src/sentry/static/sentry/app/components/forms/selectControl.jsx#L31

Might be easier to use the height now.